neptune/__init__.py

```diff
@@ -73,12 +75,21 @@ def create_experiment(name=None,
     global project
     if project is None:
         raise Uninitialized()
+
+    # pylint: disable=bare-except
+    if git_info is None:
+        try:
```

Comment:
This try/except is unnecessary - there's already one in the implementation. If the implementation guarantees that it doesn't throw, we should rely on this guarantee in the rest of the code.

Reply:
Also, why would we call `get_git_info` in two places?